src/main/resources/languages/en/messages.yml

```diff
@@ -44,6 +44,9 @@ placeholderapi:
 
 guide:
   locked: 'LOCKED'
+  click-unlock: You Must Unlock This!
+  cost: 'Cost: '
+  level: ' Level(s)'
```

Comment:
I personally dislike the space here, I think the language file should not include this space and move the space to the code. Not sure how other feels about this one. Any suggestion on this?

Reply:
without the spaces the languagewords is combined so in the lore stands without them cost1Level(s)
So i add the Spaces for it.  

i dont know how to add spaces in the code by "&7" + Slimefun.getLocalization().getMessage(p, "guide.Cost") + research.getCost() + Slimefun.getLocalization().getMessage(p, "guide.Level"))) in line 292 at SurvivalSlimefunGuide.java

so i add spaces in the message key to solve this. Even if it's probably wrong.
I'm still pretty new.

Reply:
on my local machine i removed for testing the spaces in the message keywords... the result is watching the picture.
![2023-09-20_17 27 26](https://github.com/Slimefun/Slimefun4/assets/50948584/79fb78b3-b5eb-4078-80ca-93dfb9a47bb1)

To solve this i add spaces ...

Or for better understand how can i add spaces in the code add line 293 at SurvivalSlimefunGuide.java